synapse/models/geospace.py

```diff
@@ -390,15 +390,49 @@ def getModelDefs(self):
                     }),
                 ),
 
+                'interfaces': (
+                    ('geo:locatable', {
```

Comment:
Is there any value is breaking the address out to capture key data points even when the full mailing isnt available (e.g., city, state, postal code). A lot of data sources often return city and state information, but not the exact address. This aligns with this ticket - https://vertexproject.atlassian.net/browse/VTX-2660

Reply:
Breaking down `geo:address` into subs ( especially automatically ) is outside of the scope for this PR. We could add them later if we decide to break down subs, but parsing arbitrary addresses (especially outside of the US) is difficult bordering in impossible :)